lib/manageiq/appliance_console/cli.rb

```diff
@@ -10,6 +10,8 @@ def say(arg)
 
 module ManageIQ
 module ApplianceConsole
+  class CliError < StandardError; end
```

Comment:
This works.
Would `ConfigurationError` be easer to read?

Reply:
I like having the class error exception name match the class name. So I'd like to leave it as `CliError` as it is the error exception for the  `Cli` class. 

Please let me know if you feel strongly about changing the name.


```
...
  class CliError < StandardError; end
      
  class Cli
...
```